raiden/raiden_service.py

```diff
@@ -395,9 +412,13 @@ def stop(self) -> None:
         #
         # We need a timeout to prevent an endless loop from trying to
         # contact the disconnected client
+        if self.api_server is not None:
+            self.api_server.stop()
         self.transport.stop()
         self.alarm.stop()
 
+        if self.api_server is not None:
+            self.api_server.greenlet.join()
```

Comment:
What about possible exceptions from the greenlet? Shouldn't we use `get` instead of `join`?

Reply:
I just did what the other components do (the lines below). But you're making a point, should I change all to use `get` instead?

Reply:
If we have some other error handling in a different place, we would get the exceptions twice. But I didn't see that in a quick search. I'm always unsure about the gevent error handling outside of simple cases.